hazelcast-sql/src/test/java/com/hazelcast/sql/SqlErrorAbstractTest.java

```diff
@@ -79,125 +72,9 @@ protected void checkTimeout(boolean useClient, int dataSetSize) {
 
         populate(instance1, dataSetSize);
 
-        // Block batches from instance2 to instance1.
-        BlockingExec.Blocker blocker = new BlockingExec.Blocker();
-
-        setExecHook(instance2, exec -> {
-            if (exec instanceof MapScanExec) {
-                return new BlockingExec(exec, blocker);
-            } else {
-                return exec;
-            }
-        });
-
         // Execute query on the instance1.
-        try {
-            HazelcastSqlException error = assertSqlException(useClient ? client : instance1, query().setTimeoutMillis(100L));
-            assertErrorCode(SqlErrorCode.TIMEOUT, error);
-        } finally {
-            blocker.unblock();
-        }
-    }
-
-    protected void checkExecutionError(boolean useClient, boolean fromFirstMember) {
-        // Start two instances and fill them with data
-        instance1 = newHazelcastInstance(false);
-        instance2 = newHazelcastInstance(true);
-        client = newClient();
-
-        populate(instance1);
-
-        // Throw an error from local executor
-        String errorMessage = "Test error";
-
-        HazelcastInstance member = fromFirstMember ? instance1 : instance2;
-        HazelcastInstance target = useClient ? client : instance1;
-
-        setExecHook(member, exec -> {
-            if (exec instanceof MapScanExec) {
-                return new FaultyExec(exec, new RuntimeException(errorMessage));
-            }
-
-            return exec;
-        });
-
-        // Execute query.
-        HazelcastSqlException error = assertSqlException(target, query());
-
-        assertErrorCode(SqlErrorCode.GENERIC, error);
-        assertEquals(member.getLocalEndpoint().getUuid(), error.getOriginatingMemberId());
-        assertTrue(error.getMessage().contains(errorMessage));
-    }
-
-    protected void checkMapMigration(boolean useClient) {
```

Comment:
Maybe instead of asserting failures - which AFAIU don't happen with Jet engine and RU support - we should assert that queries complete successfully?

Reply:
To my mind, this test is redundant, because we have `MapIndexScanPMigrationStressTest` for that.
Also it's **extremely** hard to cause migrations with Jet internals, instead of IMDG's one. It's also a reason why `MapIndexScanPMigrationStressTest` is so heavyweight.